src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page {
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
```

Comment:
default property

Reply:
what are you mean ?

Reply:
font-style: normal - is default you don't need to set it